snowpack/src/hmr-server-engine.ts

```diff
@@ -41,10 +41,11 @@ export class EsmHmrEngine {
   private currentBatchTimeout: NodeJS.Timer | null = null;
   private cachedConnectErrors: Set<HMRMessage> = new Set();
   readonly port: number = 0;
+  private wss: WebSocket.Server;
 
   constructor(options: EsmHmrEngineOptions) {
     this.port = options.port || DEFAULT_PORT;
-    const wss = options.server
```

Comment:
nit: since this is now a class property, why not remove the `const` assignment?

Reply:
Just because the variable is used elsewhere in the constructor and I didn't feel like changing it :)